test/Altinn.App.Core.Tests/Altinn.App.Core.Tests.csproj

```diff
@@ -97,6 +97,7 @@
 
   <ItemGroup>
     <Watch Remove="Extensions\DictionaryExtensionsTests.cs" />
+    <Watch Remove="Infrastructure\Clients\Email\EmailNotificationClientTests.cs" />
```

Comment:
What are these for?

Reply:
'dotnet watch' will ignore the files listed here

Reply:
Why should it ignore changes to those files?

Reply:
Is there anything that would cause this file to change that should not cause a `dotnet watch test` to run tests again?

Reply:
The only reason I can think of to have it here is to avoid hot reload when running with "dotnet watch" instead of "dotnet run". It was not a change I made manually or with intent so I will remove it.

Reply:
I also just realized which csproj this is in... Yeah it makes no sense